frontend/components/Filter/OrganiserFilterDialog.tsx

```diff
@@ -0,0 +1,415 @@
+"use client";
+
+import { EventData } from "@/interfaces/EventTypes";
+import { ChevronDownIcon, ChevronUpIcon } from "@heroicons/react/24/outline";
+import { useMemo, useState } from "react";
+import Datepicker from "react-tailwindcss-datepicker";
+import ListBox from "../ListBox";
+
+export enum SortByCategory {
+  HOT,
+  TOP_RATED,
+  PRICE_ASCENDING,
+  PRICE_DESCENDING,
+  DATE_ASCENDING,
+  DATE_DESCENDING,
+}
+
+export const DEFAULT_SORT_BY_CATEGORY = SortByCategory.HOT;
+export const HOT_SORTBY_STRING = "Hot";
+export const TOP_RATED_SORTBY_STRING = "Top Rated";
+export const PRICE_ASCENDING_SORTBY_STRING = "Price Ascending";
+export const PRICE_DESCENDING_SORTBY_STRING = "Price Descending";
+export const DATE_ASCENDING_SORTBY_STRING = "Date Ascending";
+export const DATE_DESCENDING_SORTBY_STRING = "Date Descending";
+
+export const DEFAULT_SEARCH = "";
+export const DEFAULT_EVENT_STATUS = "";
+export const DEFAULT_EVENT_TYPE = "";
+export const DEFAULT_MIN_PRICE = null;
+export const DEFAULT_MAX_PRICE = null;
+export const DEFAULT_START_DATE = "";
+export const DEFAULT_END_DATE = "";
+export const defaultDateRange = {
+  startDate: DEFAULT_START_DATE,
+  endDate: DEFAULT_END_DATE,
+};
+export const DAY_START_TIME_STRING = " 00:00:00";
+export const DAY_END_TIME_STRING = " 23:59:59";
+export const EMPTY_LOCATION_STRING = "";
+
```

Comment:
Would it be possible to move these constant exported variables to a central location along with the other filter file in sportshub? We preferably do not want to have repeated constant variables throughout the code base.

Reply:
it could be but i dont see where else we would use this other than our normal dashboard but shouldn't be too hard to do